utils/node_local_test.py

```diff
@@ -81,6 +82,17 @@ def set_args(self, args):
         """Set command line args"""
         self.args = args
 
+        # Parse the max log size.
+        if args.max_log_size:
+            size = args.max_log_size
+            if size.endswith('MiB'):
+                size = int(size[:-3])
+                size *= (1024 * 1024)
+            elif size.endswith('GiB'):
+                size = (size[:-3])
+                size *= (1024 * 1024 * 1024)
+            self.max_log_size = size
+
```

Comment:
Do we want some sanity checks here?
What if `size` doesn't end with `MiB` or `GiB`?
What if `size` isn't a number at all?
What if `size` is something like `0MiB` or `1MiB`?

Reply:
Added int where needed.  the last int will crash if any other suffix or non-numerical input is provided.